### DISCUSSION TITLE
Administrative Units and Borders have inconsistent Geometry

### DISCUSSION THREAD

**Discussion by MayteTDGeograma** [gopeg/issues-barriers#29] [General]
Data has some inconsistencies in the borders. Data providers should explore their neighbour’s data to check geometrical inconsistencies and coordinate with them to solve discrepancies.

It would be helpful to inform about the country-specific administrative structure or organisation of each dataset to understand better the specific characteristics of the data. Each country has some specific situations that are not obvious. (Cities as regions, a unique type of units, non-continuous levels, etc) It would be recommended to use the “lineage” in the metadata to include this useful information.

Each administrative hierarchical level should be accompanied by the name as foreseen in the INSPIRE data model with the attribute “NationalLevelName”. Despite being a voidable attribute (see “POPIMPACT_Non-mandatory attributes”), it is very useful to understand the data properly. It would be recommended, as well, to include the language attribute in this tag.

**Comment by AndersOstmanNovogit** [comment]
The question about inconsistencies between data sets is reported in section 4.2.9 in D4.3.
There is a variety of standard GIS tools to detect and also correct inconsistencies. If possible, adjustments may be carried out on the border itself or and/or on cross-border features such as roads. One problem is to backpropagate such changes into the original data bases, specifically if we are dealing with administrative borders.
@MayteTDGeograma : Which solution did you use?

The question about insufficient documentation is addressed in #30. I have added a text about FAQ's to the potential solutions in D4.3.

**Reply by AndersOstmanNovogit** [reply]
Moved to D4.3 section 4.2.4
